### PR TITLE
Hardcode default card summary

### LINKED ISSUE #85: MS teams card not having summary field if Alertmanager json has multiple alerts
We observed that if AlertManager aggregates multiple alerts, MS teams card generated does not contain summary field and causing Teams to fail with Bad Request error.

`time="2020-01-23T14:18:34Z" level=info msg="Microsoft Teams response text: Summary or Text is required."
time="2020-01-23T14:18:34Z" level=error msg="Failed sending to the Teams Channel. Teams http response: 400 Bad Request"`

Please review.

### PR DESCRIPTION
**Pull request by Knappek**
fixes #85

### CODE REVIEW

**Comment by AndreasKnapp1812** [comment]
@bzon can we merge this?

**Comment by bzon** [comment]
> @bzon can we merge this?

I have a request to not change the default server bind address 👍

File: Makefile [hunk 1]
```diff
@@ -64,7 +64,7 @@ docker-hub-push: docker
 	docker push $(DOCKER_HUB_REPO):$(VERSION)
 
 run-osx: dep darwin
-	bin/prometheus-msteams-darwin-amd64 server $(RUN_ARGS)
+	bin/prometheus-msteams-darwin-amd6 -http-addr localhost:2000 $(RUN_ARGS)
```
Comment (bzon) [inline Makefile:67]: Typo in `amd6` here. By the way, would it be better if this is just `go run cmd/server/main.go` instead?

Comment (Knappek) [reply Makefile]: @bzon I changed it to a target named `local` which seems to make more sense to me.

**Pull request merged**